processor/lsmintervalprocessor/internal/merger/model.go

```diff
@@ -406,3 +417,125 @@ func mergeDeltaHistogramDP(from, to pmetric.HistogramDataPoint) {
 		toCounts.SetAt(i, fromCounts.At(i)+toCounts.At(i))
 	}
 }
+
+func mergeDeltaExponentialHistogramDP(from, to pmetric.ExponentialHistogramDataPoint) {
+	if to.Count() == 0 {
```

Comment:
We should short-circuit these 2 conditions by just copying the whole DO. I haven't done this because I wanted to confirm if zero count is included in count but then it slipped my mind. I will fix this

Reply:
This is done!